src/Library/demos/Video/main.py

```diff
@@ -0,0 +1,23 @@
+import gi
+
+gi.require_version("Gtk", "4.0")
+gi.require_version("Gio", "2.0")
+from gi.repository import Gtk, Gio
+import workbench
+
+video: Gtk.Video = workbench.builder.get_object("video")
+video.set_file(Gio.File.new_for_uri(workbench.resolve("./workbench-video.mp4")))
+
+
+def on_pressed(*_):
```

Comment:
`*_` is a bit odd style, normally this is always called `*args`, but it's OK.

Reply:
But yeah this is kind of the way to go if you don't care about the signal's parameters.

Reply:
I know we already merged a few PRs with `*_` but I would really prefer `*args`.

@sonnyp I'll do that later once the storm of demo PRs is done, I'll also see if we can add linting then, it would probably point this out.

Reply:
In the first case, yes. In the second you  could do `x, _a, y, _b` if it's just one extra argument or also `x, _a, y, *args` in general.